tests/sympc/approximations/reci_test.py

```diff
@@ -22,5 +22,15 @@ def test_reciprocal(method, get_clients) -> None:
     x_reciprocal = reciprocal(x, method=method)
     assert torch.allclose(x_secret_reciprocal, x_reciprocal.reconstruct(), atol=1e-1)
 
+
+def test_exception(get_clients) -> None:
```

Comment:
MAYBE: Could you also add the exception name to the test?
like: ```test_exception_value_error```?

Reply:
Sure